Marlin/Configuration.h

```diff
@@ -582,7 +582,7 @@
  *
  */
 #if EITHER(KNUTWURST_4MAX, KNUTWURST_4MAXP2)
-  #define TEMP_SENSOR_0 1
+  #define TEMP_SENSOR_0 11
```

Comment:
Afaik the default value for the 4MAX Pro 2.0 is 1 and not 11. If you want a separate configuration for 4MAX Pro v1, please add a new environment variable

Reply:
I will make a new environment for the 4MaxPro1. This changes are made for 4MaxPro1 on the profile of 4MaxPro2.